lib/board.rb

```diff
@@ -0,0 +1,74 @@
+require 'pry'
+
+class Board
+
+attr_accessor :cells
+
+@cells = []
```

Comment:
Initializing `@cells` outside of an instance method context makes this a 'class instance variable' instead of a typical instance variable, which is probably not what you intended here. Class instance variables are only accessible through class methods. Instead, initialize your variable in the `initialize` or `reset` method.

See this [Ruby Monk explanation](https://rubymonk.com/learning/books/4-ruby-primer-ascent/chapters/45-more-classes/lessons/113-class-variables#magic_block-4477) for more details.